Microsoft.Toolkit.Services/Services/MicrosoftGraph/MicrosoftGraphAuthenticationHelper.cs

```diff
@@ -190,10 +191,21 @@ public bool Logout()
         /// </summary>
         /// <returns>Success or failure</returns>
         public bool LogoutV2()
+        {
+            Task<bool> task = Task.Run<bool>(async () => await LogoutV2Async());
+            return task.Result;
```

Comment:
I'm assuming we can't get around this, worried about DeadLocking though, but solution is to use other method... @azchohfi any thoughts if we're concerned here or just use as another forcing function to use Async method?